circle.yml

```diff
@@ -8,6 +8,7 @@ macBuildFilters: &macBuildFilters
     branches:
       only:
         - develop
+        - tgriesser/build/root-yarn-install
```

Comment:
I assume you want to remove your branches before you merge?

Reply:
Yep, though I can't remove yet since it's needed to run the full tests.

It's not a huge deal since it only has an effect if it's that branch - usually they just get cleaned up later when someone needs to add a new branch in (the 9.0 branch is still hanging around in develop): 

https://github.com/cypress-io/cypress/blob/002fbdda600aa83fce90dfa2d687bbf8aae6aa87/circle.yml#L6-L11